doc/api/process.md

```diff
@@ -97,6 +97,60 @@ the child process.
 The message goes through serialization and parsing. The resulting message might
 not be the same as what is originally sent.
 
+### Event: 'multipleResolves'
+<!-- YAML
+added: REPLACEME
+-->
+
+* `type` {string} The error type. One of `'resolveAfterResolved'` or
+  `'rejectAfterResolved'`.
+* `promise` {Promise} The promise that resolved or rejected more than once.
+* `value` {any} The value with which the promise was either resolved or
+  rejected after the original resolve.
+* `message` {string} A short description of what happened.
+
+The `'multipleResolves'` event is emitted whenever a `Promise` has been either:
+
+* Resolved more than once.
+* Rejected after resolve.
+
+This is useful to track errors in your application while using the promise
+constructor. Otherwise such mistakes are silently swallowed due to being in a
+dead zone.
+
+It is recommended to end the process on such errors, since it is unclear what
+the actual intention was and the process could be in an undefined state.
```

Comment:
I would just cut `it is unclear what the actual intention was and` — I feel like it makes it more verbose and doesn't really add anything.